system/Test/Mock/MockCodeIgniter.php

```diff
@@ -15,6 +15,16 @@
 
 class MockCodeIgniter extends CodeIgniter
 {
+    /**
+     * Context
+     *  web:     Invoked by HTTP request
+     *  php-cli: Invoked by CLI via `php public/index.php`
+     *  spark:   Invoked by CLI via the `spark` command
+     *
+     * @phpstan-var 'php-cli'|'spark'|'web'
+     */
```

Comment:
IDEs will use the parent docblock so it's fine to drop this.

Reply:
Dropped.